src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -54,25 +54,25 @@ export default {
  the selected dataset snapshots.'),
 
     encryption_placeholder: T('Encryption'),
-    encryption_tooltip: T(''),
+    encryption_tooltip: T('Checking this box, enables additional options for encryption replication'),
```

Comment:
Cleanup suggestion:
"Set to use encryption when replicating data. Additional encryption options will appear."